action/edit.go

```diff
@@ -69,7 +70,10 @@ func (s *Action) editor(content []byte) ([]byte, error) {
 		return []byte{}, fmt.Errorf("failed to create tmpfile to start with %s: %v", editor, tmpfile.Name())
 	}
 
-	cmd := exec.Command(editor, tmpfile.Name())
+	cmdArgs := strings.Split(editor, " ")
```

Comment:
@metalmatze 
I changed it accordingly and because https://godoc.org/github.com/kballard/go-shellquote#Split returns an error I added the error message ``failed to parse EDITOR command `%s` `` if error not nil.
So for an given invalid command, eg `EDITOR="subl '"`, the error message is 
``Error: failed to parse EDITOR command `subl '` ``.
Do you like this approach?